CHANGELOG.rst

```diff
@@ -9,6 +9,10 @@ Changelog
   If you wish to have the old behavior, set ``ordering=[]`` to your ``TaggableManager`` instance.
   We believe that this should not cause a noticable performance change, and the number of queries involved should not change.
 
+5.0.2 (2024-06-19)
+~~~~~~~~~~~~~~~~~~
```

Comment:
We don't need this part, we'll make a release header once we put a release out that has that

Reply:
I removed this part on my end this time around